### ISSUE TITLE
Add scanner screen

### ISSUE THREAD

**Issue by bonomat** [get10101/10101#59]

**Comment by da-kami** [comment]
I added a scanner screen as part of https://github.com/get10101/10101/issues/58

I have only modeled the scanner in context of `Send` so far, both for Lightning and Bitcoin; assuming that the scanner will be able to detect a lightning invoice and a bitcoin address.
Note that I added "quick action" buttons to the scanner screen - those are not fully modeled out, I would 

@bonomat how far do we want to go with the scanner? 
I suppose LN-URL (...) is not part of the MVP scope? 

From my point of view what is modeled should be enough to implement the scanner screen, so I think we can close this ticket!
